src/ui/public/field_editor/field_editor.js

```diff
@@ -99,11 +99,12 @@ uiModules
               });
           }
           const confirmModalOptions = {
-            confirmButtonText: 'Delete field',
-            onConfirm: doDelete
+            confirmButtonText: 'Delete',
+            onConfirm: doDelete,
+            title: `Delete field ${self.field.name}?`
```

Comment:
@gchaps Same idea here regarding quotation marks.

Reply:
Good point, @cjcenizal 

Our writing guidelines say to avoid unnecessary punctuation unless it helps clarify meaning, which in this case it does.  I did some research and I noticed that others do use quotation marks around a field name in a title.  For example:

Delete file “New Text Document”

If this looks too heavy, we can also try single quotation marks:

Delete file ‘New Text Document’

Or if you don’t want punctuation in the title, we can use a title followed by a sentence.  For example,

Delete this file?

The file ‘New Text Document’ will be moved to the Recycle bin.

My preference is to use single quotation marks.

Reply:
Single quotations marks sounds good to me. What do you think @nreese ?

Reply:
@gchaps Can you also update the writing guidelines with this rule?

Reply:
@cjcenizal Yes, good idea to add this to the guidelines.